mempool/clist_mempool.go

```diff
@@ -390,7 +390,7 @@ func (mem *CListMempool) isFull(txSize int) error {
 		txsBytes = mem.SizeBytes()
 	)
 
-	if memSize >= mem.config.Size || int64(txSize)+txsBytes > mem.config.MaxTxsBytes {
+	if memSize >= mem.config.Size || uint64(txSize)+uint64(txsBytes) > uint64(mem.config.MaxTxsBytes) {
```

Comment:
wouldn't this also cause some overflow issues? https://go.dev/play/p/TqOVImhtHNm

```
package main

import (
	"fmt"
	"math"
)

func main() {
	var maxTxsBytes = uint64(1000)        // max txs bytes
	var txsBytes = uint64(math.MaxUint64) // max value = 18446744073709551615
	var txSize = uint64(1)
	fmt.Println(txsBytes + txSize)             // output: 0 (overflow!)
	fmt.Println(txsBytes+txSize > maxTxsBytes) // false: should be true
}
```
outputs
```
0
false
```

Reply:
It's the same, yeah. Using uint64 merely reduces the chances of that happening. The real solution is to use safe math

Reply:
@andynog could you open an issue, against main, to replace it with safeAdd?

Reply:
OK, I'll add an issue.

Reply:
https://github.com/cometbft/cometbft/issues/2537